heartbeat/aws-vpc-move-ip

```diff
@@ -363,7 +385,13 @@ ec2ip_get_and_configure() {
 
 	# Reconfigure the local ip address
 	ec2ip_drop
-	cmd="ip addr add ${OCF_RESKEY_ip}/32 dev $OCF_RESKEY_interface"
+
+	extra_opts=""
+	if [ -n "$OCF_RESKEY_iflabel" ]; then
+		extra_opts="$extra_opts label $OCF_RESKEY_interface:$OCF_RESKEY_iflabel"
+	fi
+
+	cmd="ip addr add ${OCF_RESKEY_ip}/32 dev $OCF_RESKEY_interface$extra_opts"
```

Comment:
This will expand to `$OCF_RESKEY_interface$OCF_RESKEY_interface$extra_opts`, and you probably want to use it in the other actions as well.

So I think a better solution is to set `interface_opt` like we do with region, and use that instead of `$OCF_RESKEY_interface` in the rest of the code:
https://github.com/ClusterLabs/resource-agents/blob/273c014a839a85ae6cd0e84f42ddced4339f37f0/heartbeat/aws-vpc-move-ip#L62=

Reply:
With `iflabel` set, the line should expand into `... dev $OCF_RESKEY_interface label $OCF_RESKEY_interface:$OCF_RESKEY_iflabel`, e.g. with `interface=eth0` and `iflabel=myvip`: `... dev eth0 label eth0:myvip`.

It should be sufficient to use the label only for the `ip address add` command. My idea was to match IPaddr2 which does the same.

Reply:
Updated patch adds the mentioned space.